apps/sms/index.html

```diff
@@ -113,6 +113,7 @@
     <script defer src="js/contact_renderer.js"></script>
     <script defer src="js/information.js"></script>
     -->
+    <link rel="import" href="waiting-screen.html">
```

Comment:
I'm not sure how `rel=import` works currently in Gaia. My impression is that it's been tailored for Settings use: 
https://github.com/mozilla-b2g/gaia/blob/ee4941acf3f4b544aa89871cb46ff306e488c493/apps/settings/index.html#L118-L213 and lines like https://github.com/mozilla-b2g/gaia/blob/ee4941acf3f4b544aa89871cb46ff306e488c493/apps/settings/index.html#L240-L241

They get replaced by the code https://github.com/mozilla-b2g/gaia/blob/ee4941acf3f4b544aa89871cb46ff306e488c493/shared/js/html_imports.js (called by https://github.com/mozilla-b2g/gaia/blob/ee4941acf3f4b544aa89871cb46ff306e488c493/shared/js/lazy_loader.js#L46-L51)

This is quite complex and it does not do what we need, I think.

I think we need to load html from an external file, using XHR (this is what https://github.com/mozilla-b2g/gaia/blob/ee4941acf3f4b544aa89871cb46ff306e488c493/shared/js/html_imports.js#L50 does). Then we could reuse the template.js https://github.com/mozilla-b2g/gaia/blob/master/shared/js/template.js library to create a template from an external file?

`Template.fromURL` would return a promise to a template object. This could be cached so that requesting the same URL twice would be fast.

What do you think ?